plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintCheckTask.kt

```diff
@@ -22,7 +22,9 @@ import org.gradle.api.tasks.PathSensitivity
 import org.gradle.api.tasks.SkipWhenEmpty
 import org.gradle.api.tasks.SourceTask
 import org.gradle.api.tasks.TaskAction
+import org.gradle.internal.reflect.JavaReflectionUtil
```

Comment:
Fair point, I'll use the reflection API directly then. 👍

Reply:
We might be able to get rid of reflection in favor of some static helper class once the plugin is built using Gradle >= 5.0.

Reply:
After seeing this PR, I needed reflection because of a breaking change in AGP and used this class. https://github.com/fourlastor/pickle/pull/22

But then Gradle 5.2 removed this class completely. That's why I've made this change to not depend on internal Gradle api. https://github.com/fourlastor/pickle/pull/27